.travis.yml

```diff
@@ -13,14 +13,15 @@ env:
     - CATKIN_LINT_ARGS='--ignore description_boilerplate --ignore target_name_collision'
     - CMAKE_ARGS=-DCMAKE_BUILD_TYPE=Release
     - NOT_TEST_DOWNSTREAM=true
-    - PYLINT_ARGS="--output-format=parseable --errors-only --ignored-modules=catkin_pkg,pylab"
+    - PYLINT_ARGS="--output-format=parseable --errors-only --ignored-modules=catkin_pkg,pylab,setuptools"
     - PYLINT2_CHECK=true
     - PYLINT3_CHECK=true
     - ROS_REPO=ros
     - VERBOSE_OUTPUT=false
     - VERBOSE_TESTS=false
   matrix:
     - ROS_DISTRO=melodic
+    - ROS_DISTRO=noetic UPSTREAM_WORKSPACE='.travis.rosinstall.noetic -cob_command_tools/cob_command_gui -cob_command_tools/cob_command_tools' CATKIN_LINT_ARGS='--ignore description_boilerplate --ignore target_name_collision --ignore unknown_package' PYLINT2_CHECK=true PYLINT3_CHECK=false  # noetic is python3 only, i.e. only pylint command exists which is used as executable in PYLINT2_CHECK
```

Comment:
`--ignore unknown_package` is due to `catkin-lint` to support package format 3 only after version `1.6.9` (see https://github.com/fkie/catkin_lint/issues/89)